src/core/Context.py

```diff
@@ -64,11 +69,11 @@ async def prompt(self, message, title=None, delete_after=True):
 
         try:
             if delete_after:
-                await msg.delete()
+                await msg.delete(delay=0)
```

Comment:
Giving `delay=0` is safe, it ignore all `HTTPExceptions`

Reply:
didn't know this either :pain: